src/pages/all-about-carbon-for-ibm.com/what-is-carbon-for-ibm.com.mdx

```diff
@@ -0,0 +1,54 @@
+<PageDescription>
+
+Carbon for IBM.com is the open source design system for IBM.com's digital experiences. The system consists of working code, design tools, and resources and is tailored to IBM.com website page makers.
+
+</PageDescription>
+
+## What is Carbon for IBM.com?
+
+As IBM.com's official design system, Carbon for IBM.com serves a wide range of designers and
+developers that are building digital products and experiences.
+
+Carbon for IBM.com is building for the IBM.com maker's business needs. It also means we have a brilliant team of engineers
+and designers working full time to help make IBM.com better. It is maintained by the Digital Design Squad, a
+core team of designers, developers, and writers. 
+
+The goals of the Carbon for IBM.com include improving visual consistency and quality for all IBM.com website pages,
+establishing a shared user-centered experience for all IBM.com visitors, and providing
+clear, discoverable guidance around design and development best practices to all Carbon for IBM.com adopters.
+
+## What is Carbon?
+
+Carbon Design System, (or simply Carbon), represents a way to implement the IBM Design Language across IBM products. Choosing Carbon as the strategic solution for Carbon for IBM.com ensures IBM has one consolidated design system across product and web. Carbon for IBM.com works alongside Carbon to produce themes, components, patterns, and guidelines to leverage IDL for IBM.com’s more expressive nature.
+
+
+## What are the key differences between Carbon and Carbon for IBM.com?
+
+**Carbon for IBM.com is for IBM.com experience.** 
+The Carbon Design System is for product experience.
+
+**Carbon for IBM.com adopters are the IBM.com website's page makers.** 
+The Carbon Design System adopters are the IBM product teams.
+
+**Carbon for IBM.com is built with <a href="https://ibmdotcom-react.mybluemix.net/?path=/story/overview-getting-started--page" target="_blank">React</a> and <a href="https://ibmdotcom-web-components.mybluemix.net/?path=/story/overview-getting-started--page" target="_blank">Web Components</a>.** 
+The Carbon Design System is built with React first and supports core parts of the system in Vanilla JS, Angular, Vue, and Svelte.
+
+**Carbon for IBM.com adopters craft IBM.com experiences by using the [Carbon Expressive kit](https://ibmdotcom-library.s3-web.us-south.cloud-object-storage.appdomain.cloud/deploy-previews/922/designing#Get-the-kit).** Carbon adopters craft product experiences by using the <a href="https://www.carbondesignsystem.com/designing/kits/sketch/" target="_blank">Carbon kit</a>.
+
+**Carbon for IBM.com is developed and maintained by the Digital Design Squad.** 
+The Carbon Design System is developed and maintained by the Carbon Design Team.
+
+
+## Questions
+
+If you are not sure which design system your team should adopt, please reach out to us on the <a href="https://cognitive-app.slack.com/archives/C2PLX8GQ6" target="_blank">#carbon-for-ibm-dotcom</a> Slack channel. 
```

Comment:
> If you are not sure which design system your team should adopt

While it is a popular question, it might create more anxiety as people could think we are exclusive. I think we should convey: 

> Carbon for IBM.com is build with Carbon at its foundation, with additional components and styling optimized for reading technical and promotional content.

Reply:
Love it!